ark/analysis/visualize.py

```diff
@@ -276,3 +277,134 @@ def visualize_neighbor_cluster_metrics(neighbor_cluster_stats, dpi=None, save_di
     # save if desired
     if save_dir is not None:
         misc_utils.save_figure(save_dir, "neighborhood_cluster_scores.png", dpi=dpi)
+
+
+def visualize_topic_eda(data, metric="gap_stat", gap_sd=True, k=None, dpi=None, save_dir=None):
+    """Visualize the exploratory metrics for spatial-LDA topics
+
+    Args:
+        data (dict):
+            The dictionary of exploratory metrics produced by
+            :func:`~ark.spLDA.processing.compute_topic_eda`.
+        metric (str):
+            One of "gap_stat", "inertia", "silhouette", "percent_var_exp", or "cell_counts".
+        gap_sd (bool):
+            If True, the standard error of the gap statistic is included in the plot.
+        k (int):
+            References a specific KMeans clustering with k clusters for visualizing the cell count
+            heatmap.
+        dpi (float):
+            The resolution of the image to save, ignored if save_dir is None
+        save_dir (str):
+            Directory to save plots, default is None
+    """
+    valid_metrics = ["gap_stat", "inertia", "silhouette", "percent_var_exp", "cell_counts"]
+    misc_utils.verify_in_list(actual=[metric], expected=valid_metrics)
+    featurization = data["featurization"]
+    data_k = {k: v for k, v in data.items() if k != "featurization"}
+    df = pd.DataFrame.from_dict(data_k)
+    df['num_clusters'] = df.index
+
+    if metric == "gap_stat":
+        if gap_sd:
+            plt.plot()
+            plt.errorbar(x=df["num_clusters"], y=df["gap_stat"], yerr=df["gap_sds"])
+        else:
+            sns.relplot(x=df["num_clusters"], y=df["gap_stat"])
+        plt.xlabel("Number of Clusters")
+        plt.ylabel("Gap")
+    elif metric == "inertia":
+        sns.relplot(x=df["num_clusters"], y=df["inertia"], kind="line")
+        plt.xlabel("Number of Clusters")
+        plt.ylabel("Inertia")
+    elif metric == "silhouette":
+        sns.relplot(x=df["num_clusters"], y=df["silhouette"], kind="line")
+        plt.xlabel("Number of Clusters")
+        plt.ylabel("Silhouette Score")
+    elif metric == "cell_counts":
+        if k is None:
+            raise ValueError("Must provide number of clusters for k value.")
+        sns.heatmap(data["cell_counts"][k], vmin=0, square=True, xticklabels=True,
+                    yticklabels=True, cmap="viridis")
+        plt.xlabel("KMeans Cluster Label")
+        if featurization == "cluster":
+            plt.ylabel("Cell Cluster")
+        elif featurization == "marker" or featurization == "avg_marker":
+            plt.ylabel("Channel Marker")
+        else:
+            plt.ylabel("Cell Counts")
+    else:
+        sns.relplot(x=df["num_clusters"], y=df["percent_var_exp"] * 100, kind="line")
+        plt.xlabel("Number of Clusters")
+        plt.ylabel("% of Total Variance Explained")
```

Comment:
```suggestion
        plt.ylabel("%% of Total Variance Explained")
```

Reply:
`%` is by default a formatting character so Python will complain it's a stray if you don't escape it.